src/cmdline/mod.rs

```diff
@@ -479,4 +486,18 @@ mod tests {
         assert!(cl.insert_multiple("foo", &["bar", "baz"]).is_ok());
         assert_eq!(cl.as_str(), "foo=bar,baz");
     }
+
+    #[test]
+    fn test_partial_eq() {
+        let mut c1 = Cmdline::new(20);
+        let mut c2 = Cmdline::new(20);
```

Comment:
Can we add a test for different max sizes, but the same string?

Reply:
Oh they were supposed to be different (i.e. 20 and 30), I've fixed this now.